Source/towners.cpp

```diff
@@ -682,6 +682,8 @@ void ProcessTowners()
 		case TOWN_COWFARM:
 			TownCowFarmer();
 			break;
+		case NUM_TOWNER_TYPES:

```

Comment:
```suggestion
		case default:
```

Reply:
I would prefer getting an error with the missing towner at build time.